platform_it/src/test/java/com/splicemachine/test/SpliceTestPlatformConfig.java

```diff
@@ -186,15 +186,18 @@ public static Configuration create(String hbaseRootDirUri,
         //
         // Threads, timeouts
         //
-        config.setLong("hbase.rpc.timeout", MINUTES.toMillis(2));
+        config.setLong("hbase.rpc.timeout", MINUTES.toMillis(5));
         config.setInt("hbase.client.max.perserver.tasks",50);
         config.setInt("hbase.client.ipc.pool.size",10);
         config.setInt("hbase.rowlock.wait.duration",0);
 
         config.setLong("hbase.client.scanner.timeout.period", MINUTES.toMillis(2)); // hbase.regionserver.lease.period is deprecated
         config.setLong("hbase.client.operation.timeout", MINUTES.toMillis(2));
-        config.setLong("hbase.regionserver.handler.count", 100);
-        config.setLong("hbase.regionserver.metahandler.count", 100);
+        config.setLong("hbase.regionserver.handler.count", 10);
```

Comment:
This will reduce the threads, thus reduce the memory usage. Since by default, each thread will use 1M memory. After the modification, the threads reduced from more than 1000 to 500, which saves about 500M memory.